packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@contentful/experience-builder-components",
-  "version": "0.0.2-alpha.26",
+  "name": "@contentful/experiences-components-react",
+  "version": "0.0.1-alpha.25",
```

Comment:
you will probalby need to update this right before the final merge into develop, but the alpha version should match what is currently in published to npm